### DISCUSSION TITLE
I love using the GraphQL / Prisma / Apollo / Postgres stack, but love the GUI of pocketbase for clients to use as a CMS

### DISCUSSION THREAD

**Discussion by oscadev** [pocketbase/pocketbase#3294] [Q&A]
Is there a way to simply use pocketbase as a GUI for my existing tech stack, for non-tech people to be able to manage content (not create new models / types, just CRUD documents in the models created by me). If so, what would be the best way? Would it be pointing Prisma to the db created by pocketbase, or would it be to somehow point pocketbase to my postgres db?

**Comment by ganigeorgiev** [comment]
PocketBase can be used only with SQLite, there is no support at the moment for other databases.

Technically it is possible external applications to work with the same SQLite db file, but the usage and experience will greatly depend on how all connected applications interact with it (eg. whether all applications can write to the file, how long a transaction lock is held, what pragmas are used when opening the connections, etc.).

In general I'd advice against using it this way, unless you really know what you are getting into.
While the Admin UI can be used as a very simple CMS it wasn't really designed with that in mind and lacks a lot of features that usually you'll find in more specialized solutions like Strapi, Directus, Statamic, etc.
If you want only an interface for managing the CRUD of your data, it might be easier to create your own GUI (_or use a template_) with whatever stack you feel more familiar working with.
